xsd/pmo/people.xsd

```diff
@@ -53,6 +53,9 @@ SOFTWARE.
           </xs:restriction>
         </xs:simpleType>
       </xs:element>
+      <xs:element name="reputation" type="xs:integer" minOccurs="0"/>
```

Comment:
@g4s8 why `minOccurs=0`? I think we have to have `minOccurs="1" maxOccurs="1"` and then create an "upgrade" that will set it all to zero for all users.